### DISCUSSION TITLE
Cursed Reinscriptions after final reinscription?

### DISCUSSION THREAD

**Discussion by BitTogether** [ordinals/ord#2373] [Ideas]
Someone had mentioned that maybe there should be a way to stop future re-inscriptions, "finalizing" a satoshi.  Although I am unsure about implementing something that limits the basic human freedom to reinscibe our sats, this function could create a new class of ordinals called "Cursed Re-Inscriptions"  that do not display after a satoshi is sealed, yet still exist onchain and conform to the other rules of the protocol.  Only the holder of a sat could create a cursed re-inscription on a sat by putting new inscriptions after some finalization process.

**Comment by stet** [comment]
Yes, I was discussing this EOF idea here - https://twitter.com/sull/status/1695528817962414089
We should discuss here, thanks for bringing it up.
